litmus-portal/graphql-server/graph/analytics.graphqls

```diff
@@ -108,15 +121,45 @@ input promQueryInput {
     minstep: Int!
 }
 
-type promResponse {
+type metricsPromResponse {
     queryid: String!
-    legends: [[String]]
-    tsvs: [[timeStampValue]]
+    legends: [String]
+    tsvs: [[metricsTimeStampValue]]
+}
+
+type metricsTimeStampValue{
+    date: Float
```

Comment:
May we change the `date: Float` to something else

Reply:
This is to solve the 2038 problem with 32bit integer which graphql supports and mitigate frontend parsing, so we have to use float which is acceptable by visx graphs as tested. @whitetiger1399 

Ref:
https://en.wikipedia.org/wiki/Year_2038_problem
https://graphql.org/learn/schema

<img width="873" alt="Screenshot 2021-05-12 at 11 00 45 AM" src="https://user-images.githubusercontent.com/50652677/117923244-5661d480-b311-11eb-94bb-17832536bb22.png">

Reply:
let it be `float` due to 32bit issue in 2k38